tests/factories.py

```diff
@@ -101,3 +102,16 @@ class Meta:
         model = restaurant_models.Floor
         sqlalchemy_session = DB.session
         sqlalchemy_session_persistence = "commit"
+
+
+class ReservationFactory(factory.alchemy.SQLAlchemyModelFactory):
+    num_of_persons = factory.Faker("pyint")
+    comment = factory.Faker("text")
+    start_time = factory.Faker("date")
+    end_time = factory.Faker("date")
```

Comment:
@vladarefiev I think we shouldn't use ``Faker`` for start/end dates because it might choose start >= end, which wouldn't be correct, right?

Reply:
@amidatonga You are correct, but it's possible if  needed to do like that:
`factories.ReservationFactory(end_date=..., ...)`

Reply:
@vladarefiev yes, but won't your tests fail from time to time, inexpectedlly? If this test sets start > end, then it will fail when inserting into the DB, I think.

Reply:
@amihaiemil done. I found a compromise ;)

Reply:
@amihaiemil by default start_time is now, end_time = now + 3 days. But if you want you can set customs dates.